Assets/Scripts/GameLobby/UI/CreateMenuUI.cs

```diff
@@ -32,6 +36,15 @@ void OnTabChanged(JoinCreateTabs tabState)
         public void SetServerName(string serverName)
         {
             m_ServerName = serverName;
+            m_CreateButton.interactable = ValidateServerName(m_ServerName) && ValidatePassword(m_ServerPassword);

```

Comment:
Added local validation of the server name and password to combat vague Validation errors from the Lobby service in this version. 

Next version will have clearer errors, and it is up to the developer to implement local validation steps like this one.